.pipelines/templates/.builder-release-template.yaml

```diff
@@ -17,6 +17,22 @@ steps:
       vstsFeedPackage: asc-baseline
       vstsPackageVersion: '1.1.0-268'
       downloadDirectory: vhdbuilder/packer
+  - task: DownloadPipelineArtifact@2
+    displayName: 'Download Kata CC UVM artifact'
+    inputs:
+      source: specific
+      project: CloudNativeCompute
+      pipeline: $(UVM_PIPELINE_ID)
+      runVersion: specific
```

Comment:
IN PROGRESS: will switch to the latest build once the target pipeline has been stabilized.

Reply:
Our new goal was to replicate the UVM build in this pipeline, right?

Reply:
We can resolve this? We are using a variable that we can specify and don't need to set an expicit ID?